enhancements/pause-and-resume-scans.md

```diff
@@ -0,0 +1,338 @@
+are going through significant changes or maintenance, pausing compliance scans
+is a more appropriate option than deleting the scan completely.
+
+From a user experience perspective, providing this functionality through an
+annotation is easier to consume than forcing users to look up `CronJob`
+rerunners for the scan their interested in and pausing it directly.
+
+## Design Details
+
+### Open Questions [optional]
+
+This is where to call out areas of the design that require closure before deciding
+to implement the design.  For instance,
+ > 1. This requires exposing previously private resources which contain sensitive
+  information.  Can we do this?
+
+
+
+### Test Plan
+
+This feature can be easily tested using our existing end-to-end test suite. We
+don't necessarily need to rely on testing the functionality of `CronJob`, but
+we can certainly test that the Compliance Operator updates the `suspend`
+attribute according to the annotation.
+
+We should have an end-to-end test that performs the following:
+
+1. Creates a new `ComplianceScan`, where the actual scan schedule doesn't necessarily matter
+2. Assert the `CronJob` associated with the `ComplianceScan` is set to `suspend=false`
+3. Pause the `ComplianceScan` using an annotation
+4. Assert the `CronJob` associated with the `ComplianceScan` is set to `suspend=true`
+5. Resume the `ComplianceScan` by removing the annotation
+6. Assert the `CronJob` associated with the `ComplianceScan` is set to `suspend=false`
+
+### Upgrade / Downgrade Strategy
+
+If applicable, how will the component be upgraded and downgraded? Make sure this
+is in the test plan.
+
+Consider the following in developing an upgrade/downgrade strategy for this
+enhancement:
+- What changes (in invocations, configurations, API use, etc.) is an existing
+  cluster required to make on upgrade in order to keep previous behavior?
+- What changes (in invocations, configurations, API use, etc.) is an existing
+  cluster required to make on upgrade in order to make use of the enhancement?
+
+Upgrade expectations:
+- Each component should remain available for user requests and
+  workloads during upgrades. Ensure the components leverage best practices in handling [voluntary
+  disruption](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/). Any exception to
+  this should be identified and discussed here.
+- Micro version upgrades - users should be able to skip forward versions within a
+  minor release stream without being required to pass through intermediate
+  versions - i.e. `x.y.N->x.y.N+2` should work without requiring `x.y.N->x.y.N+1`
+  as an intermediate step.
+- Minor version upgrades - you only need to support `x.N->x.N+1` upgrade
+  steps. So, for example, it is acceptable to require a user running 4.3 to
+  upgrade to 4.5 with a `4.3->4.4` step followed by a `4.4->4.5` step.
+- While an upgrade is in progress, new component versions should
```

Comment:
I don't understand this enough. I am concern though if the paused scan will still hold reliable results after the upgrade of a cluster. Or, on another hand, if there is an upgrade in the scan profile - how do we apply this retroactively to a paused scan. To me it still feels maybe better to delete a scan and start a new after maintenance then resume an old scan.

Reply:
I attempted to describe this is in more detail in the next revision of the patch.

Here we're specifically pausing the scan schedule, and not stopping a scan while it is in progress. Does that help clarify?